dali/test/python/test_fw_iterators.py

```diff
@@ -679,13 +745,15 @@ def create_custom_pipeline(batch_size, num_threads, device_id, num_gpus, data_pa
 
 
 def test_pytorch_iterator_feed_ndarray():
-    from nvidia.dali.plugin.pytorch import DALIGenericIterator as PyTorchIterator
     from nvidia.dali.plugin.pytorch import feed_ndarray as feed_ndarray
     import torch
     num_gpus = 1
     batch_size = 100
-    pipes, _ = create_pipeline(lambda gpu: create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu, num_gpus=num_gpus,
-                                                                  data_paths=image_data_set), batch_size, num_gpus)
+    pipes, _ = create_pipeline(
+        lambda gpu:
+        create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
+                               num_gpus=num_gpus, data_paths=image_data_set), batch_size, num_gpus
+    )
```

Comment:
```suggestion
    pipes, _ = create_pipeline(
        lambda gpu: create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
                                           num_gpus=num_gpus, data_paths=image_data_set),
        batch_size, num_gpus)
```

Reply:
... and the remaining ones in this file. Would you mind applying my suggestion to the rest of those?